src/services/repo/hooks.spec.js

```diff
@@ -272,7 +273,12 @@ describe('useActivateFlagMeasurements', () => {
   function setup() {
     server.use(
       graphql.mutation('ActivateMeasurements', (req, res, ctx) => {
-        return res(ctx.status(200), ctx.data())
+        return res(
```

Comment:
Could we add a test case for when this 404's if we don't have one already?

Reply:
yah it's missing some coverage 😿